src/Pate/Equivalence/StatePred.hs

```diff
@@ -61,12 +61,12 @@ muxStatePred sym p predT predF = case WI.asConstantPred p of
       (M.zipWithAMatched (\_ p1 p2 -> WI.baseTypeIte sym p p1 p2))
       (predRegs predT)
       (predRegs predF)
-    stack <- PEM.muxMemPred sym p (predStack predT) (predStack predF)
-    mem <- PEM.muxMemPred sym p (predMem predT) (predMem predF)
+    stack <- PEM.mux sym p (predStack predT) (predStack predF)
+    mem <- PEM.mux sym p (predMem predT) (predMem predF)
     return $ StatePred regs stack mem
 
 statePredFalse :: WI.IsExprBuilder sym => sym -> StatePred sym arch
```

Comment:
This isn't necessarily part of this PR, but what is a `False` `StatePred`? One that is totally empty and covers nothing?

Reply:
Yes it's empty, there is another branch in progress which tries to standardize some of the language (renaming `StatePred` to EquivalenceDomain and `statePredFalse` is then just `empty`)